internal/pkg/template/templates/environment/partials/custom-resources.yml

```diff
@@ -36,7 +37,6 @@ CustomDomainAction:
   Metadata:
     'aws:copilot:description': 'Add an A-record to the hosted zone for the domain alias'
   Condition: HasAliases
-  DependsOn: HTTPSCert
```

Comment:
stoopid question: why does it not need to depend on `HTTPSCert` now?

Reply:
Because it's redundant. It doesn't need to depend on the `HTTPSCert`.